prov/verbs/src/verbs_info.c

```diff
@@ -892,6 +898,7 @@ static int fi_ibv_getifaddrs(const char *service, uint64_t flags, struct fi_info
 		if (!ifa->ifa_addr || !(ifa->ifa_flags & IFF_UP) ||
 				!strcmp(ifa->ifa_name, "lo"))
 			continue;
+		// TODO call a function here that filters ifa based on interface name
```

Comment:
is'n it done in fi_ibv_rdm_find_sysaddrs call? as I can see there is filter by iface name...